components/core/src/clp_s/search/OutputHandler.cpp

```diff
@@ -23,18 +28,14 @@ NetworkOutputHandler::NetworkOutputHandler(
     }
 }
 
-void NetworkOutputHandler::write(std::string const& message, epochtime_t timestamp) {
-    msgpack::type::tuple<epochtime_t, std::string, std::string> src(timestamp, message, "");
-    msgpack::sbuffer m;
-    msgpack::pack(m, src);
-
-    if (-1 == send(m_socket_fd, m.data(), m.size(), 0)) {
-        throw OperationFailed(ErrorCode::ErrorCodeFailureNetwork, __FILE__, __LINE__);
-    }
-}
-
-void NetworkOutputHandler::write(std::string const& message) {
-    msgpack::type::tuple<epochtime_t, std::string, std::string> src(0, message, "");
+void NetworkOutputHandler::write(
+        string_view message,
+        epochtime_t timestamp,
+        string_view archive_id
+) {
+    static constexpr string_view cOrigFilePathPlaceholder{""};
+    msgpack::type::tuple<epochtime_t, string, string, string> const
+            src(timestamp, message, cOrigFilePathPlaceholder, archive_id);
```

Comment:
```suggestion
    msgpack::type::tuple<epochtime_t, string, string, string> const
            src(timestamp, message, string_view{}, archive_id);
```

Reply:
I personally would prefer to give the 3rd parameter a name, so it's clear for developers to see what has been assigned an empty string.

Reply:
Sure.